src/content/docs/accounts/accounts-billing/new-relic-one-pricing-users/usage-queries-alerts.mdx

```diff
@@ -150,7 +150,7 @@ Here are some user-related query examples. For details on how users are counted,
 
 ## Set usage alerts [#alerts]
 
-To help [manage your billable data](/docs/telemetry-data-platform/get-data-new-relic/manage-data/manage-your-data), you can set alerts to notify you of unexpected increases in usage. To learn about how to create alerts, see [Alert workflow](/docs/alerts/new-relic-alerts/getting-started/new-relic-alerts-concepts-workflow#workflow).
+To help [manage your billable data](/docs/telemetry-data-platform/get-data-new-relic/manage-data/manage-your-data), you can set alerts to notify you of unexpected increases in usage. Learn how to create alerts with [NRQL queries here](/docs/alerts-applied-intelligence/new-relic-alerts/alert-conditions/create-nrql-alert-conditions/#create).
```

Comment:
usually we tend to avoid the 'here' and try to have the link phrasing as descriptive of what's at the link; in this case I might go with 'Learn how to create NRQL alert conditions' and have the whole thing be a link.